### ISSUE TITLE
KSQL server start main class not found

### ISSUE THREAD

**Issue by HariPandey8** [confluentinc/ksql#2135]
Hi,
While trying to ksql-server-start --help getting below error. I m trying to start through wsl.

Could not find or load main class io.confluent.ksql.rest.server.KsqlServerMain

Logs:
if [ $# -lt 1 ];
then
    echo "USAGE: $0 [-daemon] ksql-server.properties"
    exit 1
fi
+ '[' 1 -lt 1 ']'

base_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )
 cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd
 dirname "${BASH_SOURCE[0]}"
+++ dirname ./bin/ksql-server-start
++ cd ./bin/..
++ pwd
+ base_dir=/mnt/c/kafka-poc/ksql-6.0.0-beta181011024526/ksql-6.0.0-beta181011024526
: "${KSQL_CONFIG_DIR:="$base_dir/config"}"
+ : /mnt/c/kafka-poc/ksql-6.0.0-beta181011024526/ksql-6.0.0-beta181011024526/config

: "${KSQL_LOG4J_OPTS:=""}"
+ : ''
if [ -z "$KSQL_LOG4J_OPTS" ] && [ -e "$KSQL_CONFIG_DIR/log4j-silent.properties" ]; then
  export KSQL_LOG4J_OPTS="-Dlog4j.configuration=file:$KSQL_CONFIG_DIR/log4j-rolling.properties"
fi
+ '[' -z '' ']'
+ '[' -e /mnt/c/kafka-poc/ksql-6.0.0-beta181011024526/ksql-6.0.0-beta181011024526/config/log4j-silent.properties ']'
+ export KSQL_LOG4J_OPTS=-Dlog4j.configuration=file:/mnt/c/kafka-poc/ksql-6.0.0-beta181011024526/ksql-6.0.0-beta181011024526/config/log4j-rolling.properties
+ KSQL_LOG4J_OPTS=-Dlog4j.configuration=file:/mnt/c/kafka-poc/ksql-6.0.0-beta181011024526/ksql-6.0.0-beta181011024526/config/log4j-rolling.properties

EXTRA_ARGS="-name ksql-server -loggc"
+ EXTRA_ARGS='-name ksql-server -loggc'

if [ -z "$KSQL_OPTS" ]; then
  KSQL_OPTS=-Dksql.server.install.dir=$base_dir
else
  KSQL_OPTS="${KSQL_OPTS} -Dksql.server.install.dir=$base_dir"
fi
+ '[' -z '' ']'
+ KSQL_OPTS=-Dksql.server.install.dir=/mnt/c/kafka-poc/ksql-6.0.0-beta181011024526/ksql-6.0.0-beta181011024526

export KSQL_OPTS
+ export KSQL_OPTS

# TODO: Enable specification of properties via command-line arg in the KsqlRestApplication class
# so that useful defaults for bootstrap server, port, etc. can be established here

exec "$base_dir"/bin/ksql-run-class io.confluent.ksql.rest.server.KsqlServerMain $EXTRA_ARGS "$@"
+ exec /mnt/c/kafka-poc/ksql-6.0.0-beta181011024526/ksql-6.0.0-beta181011024526/bin/ksql-run-class io.confluent.ksql.rest.server.KsqlServerMain -name ksql-server -loggc --help
Error: Could not find or load main class io.confluent.ksql.rest.server.KsqlServerMain

**Comment by cricket007** [comment]
Based on `/mnt/c/` you are using Windows?

**Comment by HariPandey8** [comment]
Yes correct I using windows wsl feature on which I m trying to setup but after trying multiple time I got success for once where all services were up and running but when I tried to restart it again then zookeeper service is up but enable start Kafka and giving error failing to start kafka

**Comment by cricket007** [comment]
I would suggest not using WSL, and rather use the `windows/*.bat` files for each respective service.

Plus, running the beta builds might not be as stable as the official releases

**Comment by cricket007** [comment]
I was referring to Kafka and Zookeeper windows scripts, actually. 

Regarding the problem, it seems to be related to #402. The alternatives being Cygwin, Docker, or just a VM
